test/sleepy_runner.rb

```diff
@@ -0,0 +1,16 @@
+require 'test_queue'
+require 'test_queue/runner/minitest'
+
+class SleepyTestRunner < TestQueue::Runner::MiniTest
+  def around_filter(*args)
+    if ENV['SLEEP_AS_RELAY'] && relay? 
+      sleep 5
+    elsif ENV['SLEEP_AS_MASTER'] && !relay?
+      sleep 5
+    end
```

Comment:
> For the sleeping to be effective I think it needs to happen before suites are assigned, i.e., before the workers send their first POP messages.

Nuts, this makes total sense. My bad.

Do we need to make changes to the runner itself to give us a place to inject this behavior for testing?